tests/utils.go

```diff
@@ -3642,3 +3647,50 @@ func WaitAgentConnected(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachin
 		return false
 	}, 420*time.Second, 2).Should(BeTrue(), "Should have agent connected condition")
 }
+
+// generatePrivateKey creates a RSA Private Key of specified byte size
+func GeneratePrivateKey(bitSize int) (*rsa.PrivateKey, error) {
+	// Private Key generation
+	privateKey, err := rsa.GenerateKey(cryptorand.Reader, bitSize)
+	if err != nil {
+		return nil, err
+	}
+
+	// Validate Private Key
```

Comment:
As far as I know, code styles recommends avoiding obvious comments, like this:

// Adding a to b
c = a + b